public/editor/scripts/analytics.js

```diff
@@ -0,0 +1,188 @@
+(function(global, factory) {
```

Comment:
Add a comment above this that it's based on https://github.com/mozilla/webmaker-analytics and we're using it under MPL2.0 https://github.com/mozilla/webmaker-analytics/blob/master/LICENSE